src/main/java/de/retest/recheck/printer/AttributesDifferencePrinter.java

```diff
@@ -10,8 +11,9 @@
 
 	private final AttributeDifferencePrinter delegate;
 
-	public AttributesDifferencePrinter( final IdentifyingAttributes attributes, final DefaultValueFinder finder ) {
-		delegate = new AttributeDifferencePrinter( attributes, finder );
+	public AttributesDifferencePrinter( final IdentifyingAttributes attributes, final DefaultValueFinder finder,
```

Comment:
I suggest providing an overloaded constructor that allows optionally to set the Highlighter, otherwise falling back to the `DefaultHighlighter`. This would also prevent this from introducing a breaking change. (Same for the other Printer classes)